lib/resource/resource.ex

```diff
@@ -3865,19 +3905,10 @@ defmodule AshGraphql.Resource do
                 of_type: type
               }
             }
-          },
-          %Absinthe.Blueprint.Schema.FieldDefinition{
```

Comment:
I removed this field definition since it doesn't see be used anywhere

Reply:
When you say its not used anywhere, wouldn't it have been used by the current type?

It remaps `more?` to `has_next_page`. Your current code handles that though so it should be fine to remove.

```elixir
     %Absinthe.Blueprint.Schema.FieldDefinition{


         description: "Whether or not there is a next page",
         identifier: :has_next_page,
         module: schema,
         name: "has_next_page",
         __reference__: ref(__ENV__),
         type: %Absinthe.Blueprint.TypeReference.NonNull{of_type: :boolean}
       },
```